cfy-go/main.go

```diff
@@ -34,7 +34,7 @@ func basicOptions(name string) *flag.FlagSet {
 
 	var defaultHost = os.Getenv("CFY_HOST")
 	if defaultHost == "" {
-		defaultHost = "localhost"
+		defaultHost = "40.71.194.91"
```

Comment:
Don't use real ip's

Reply:
@0lvin Yes this is while I was testing , I will remove it